### ISSUE TITLE
[Plots] Reload button doesn't respect current zoom level

### ISSUE THREAD

**Issue by jvigliotta** [nasa/openmct#7929]
#### Summary

When zoomed into a plot, clicking the Reload action will reset the zoom level.

#### Expected vs Current Behavior

Would like the zoom level to be respected and the data in the zoomed in portion of the plot to be displayed correctly (not at the same "resolution" as the time conductor range).

#### Steps to Reproduce

1. Zoom in to a plot
2. Select the Reload option from the 3-dot menu
3. Notice the plot is reloaded, but the zoom level is thrown away

#### Environment

* Open MCT Version: 
* Deployment Type: 
* OS:
* Browser:

#### Impact Check List

- [x] Data loss or misrepresented data?
- [ ] Regression? Did this used to work or has it always been broken?
- [x] Is there a workaround available? zoom back in, but not desirable
- [x] Does this impact a critical component?
- [ ] Is this just a visual bug with no functional impact?
- [ ] Does this block the execution of e2e tests?
- [ ] Does this have an impact on Performance?

#### Additional Information

Reload action in the 3-dot menu
![Screenshot 2024-11-13 at 11 13 10 AM](https://github.com/user-attachments/assets/574e374e-44b2-4622-9245-370c946690f8)

**Comment by shefalijoshi** [comment]
@charlesh88 @akhenry 
It appears that there is ambiguity in the expected outcome of the Reload action in the ... menu vs the reset zoom action.
Can we discuss this and either re-word the Reload action or decide if we want Reload to behave like Reset pan/zoom?

**Comment by akhenry** [comment]
Yeah reload is essentially like a browser refresh but restricted to the target object. It completely unloads and reloads the associated Vue component. One consequence of this is that the data is also reloaded - which is what it's used for 99% of the time.

So this begs the question - should we change the implementation of the action and make it a data reload action, rather than a view reload action? Preserving the current view state is then trivial because we're not throwing it away, we're just throwing away telemetry and re-requesting it.

**Comment by charlesh88** [comment]
I agree with this, it should work as described.
